README.md

```diff
@@ -22,8 +22,13 @@ A list of awesome resources for (tech) event organisers to help grow and improve
 ## Organising conferences
 - [Less obvious conference list](//github.com/mxsasha/lessobviouschecklist) A great checklist of things you should consider when running a conference
 
+## Guide to hosting a panel discussion
```

Comment:
Heya - could we make this title more generic like "Organising specific types of events"? Could you then also add an anchor link to the contents to the section?